.gitignore

```diff
@@ -2,4 +2,5 @@
 .DS_Store
 .vagrant
 node_modules
-Node.js.lnk
\ No newline at end of file
+Node.js.lnk
+package-lock.json
```

Comment:
package-lock should be added, it is now actually added to master to lock dependancies down, given we only ever touch this repo once a year, we should lock it to minimise dependancy failures

Reply:
Fixed.